treeherder/etl/perf.py

```diff
@@ -35,6 +35,23 @@ def _get_signature_hash(signature_properties):
     return sha.hexdigest()
 
 
+def _create_or_update_signature(repository, signature_hash, framework, defaults):
+    signature, created = PerformanceSignature.objects.get_or_create(
+        repository=repository,
+        signature_hash=signature_hash,
+        framework=framework,
+        defaults=defaults)
+    if not created:
+        if signature.last_updated > defaults['last_updated']:
+            defaults['last_updated'] = signature.last_updated
+        signature, _ = PerformanceSignature.objects.update_or_create(
+            repository=repository,
+            signature_hash=signature_hash,
+            framework=framework,
+            defaults=defaults)
```

Comment:
here we have respository=respository, but below we change from respository=repository -> just repository.